### PR TITLE
Build dependencies

### PR DESCRIPTION
**Pull request by akenmorris**
Rename superbuild.sh to build_dependencies.sh

Builds only dependencies now, but prints a cmake command at the end which can be used to build ShapeWorks.

### CODE REVIEW

File: build_dependencies.sh [hunk 1]
```diff
@@ -33,22 +33,15 @@ usage()
   echo "  -h,--help               : Show this screen."
   echo "  --clean                 : Remove all build directories and clone implicit dependencies."
   echo "                          : (note that user-specified paths such as --itk-dir=<path> will not be deleted)."
-  echo "  --no-gui                : Do not build the ShapeWorks gui applications, which require Qt."
+  echo "  --no-gui                : Do not build the dependcies with GUI support, which require Qt."
```
Comment (sheryjoe) [inline build_dependencies.sh]: Please fix typo: "dependencies"

File: build_dependencies.sh [hunk 2]
```diff
@@ -33,22 +33,15 @@ usage()
...
   echo "                          : The GUI is built by default if qmake >= $QT_MIN_VER is found in the path."
-  echo "  --with-studio           : Build ShapeWorksStudio (default off)."
-  echo "  -b,--build-dir=<path>   : Build directory for ShapeWorks and its implicit dependencies (VXL, VTK, and ITK)."
-  echo "                          : By default uses a subdirectory of the current directory called 'build'."
-  echo "  -i,--install-dir=<path> : Install directory for ShapeWorks and its implicit dependencies (VXL, VTK, and ITK)."
-  echo "                          : By default uses a subdirectory of the current directory called 'install'."
+  echo "  -b,--build-dir=<path>   : Build directory for dependencies."
+  echo "                          : By default uses a subdirectory of the current directory called 'dependencies/build'."
+  echo "  -i,--install-dir=<path> : Install directory dependencies."
```
Comment (sheryjoe) [inline build_dependencies.sh]: Missing for: Install directory "for" dependencies

File: BUILD.md [hunk 3]
```diff
@@ -93,7 +92,7 @@ Optional:
 ```
   -DBuild_Studio=[OFF|ON]             default: OFF
   -DBuild_View2=[OFF|ON]              default: OFF
-  -DBuild_Post=[OFF|ON]               default: ON
+  -DBuild_Post=[OFF|ON]               default: OFF
   -DCMAKE_INSTALL_PREFIX=<path>       default: ./install
```
Comment (sheryjoe) [inline BUILD.md]: Is ./install the default for shapeworks or the dependencies?

File: BUILD.md [hunk 4]
```diff
@@ -113,7 +112,7 @@ Add the ShapeWorks and dependency binaries to the path:
 ### Examples
 *OSX* example that builds dependencies separately, then generates an XCode project for ShapeWorks:  
 ```
-$ ./superbuild.sh  --dependencies_only --build-dir=../dependencies --install-dir=../dependencies
+$ ./build_dependencies.sh --build-dir=../dependencies --install-dir=../dependencies
```
Comment (sheryjoe) [inline BUILD.md]: ../dependencies or ./dependencies? not consistent with the usage output of build_dependencies.sh.

File: BUILD.md (hunk 3 above)
Comment (akenmorris) [reply BUILD.md]: For shapeworks, in CMakeLists.txt, we have:

```
set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/install" CACHE STRING "Install path prefix, \
prepended onto install directories.")
```

File: BUILD.md (hunk 4 above)
Comment (akenmorris) [reply BUILD.md]: I've updated it.  The default install_dir for "build_dependencies.sh" is $PWD/dependencies/install

File: build_dependencies.sh (hunk 1 above)
Comment (akenmorris) [reply build_dependencies.sh]: 👍

File: build_dependencies.sh (hunk 2 above)
Comment (akenmorris) [reply build_dependencies.sh]: 👍

**Pull request merged**